src/NosCore.Core/Controllers/ChannelController.cs

```diff
@@ -45,7 +46,7 @@ public class ChannelController : Controller
     {
         private readonly WebApiConfiguration _apiConfiguration;
         private readonly ILogger _logger;
-        private int _id;
+        private int _id = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this initialization to '_id', the compiler will do that for you.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262030)